ludwig/utils/visualization_utils.py

```diff
@@ -39,14 +39,16 @@
 import ludwig.contrib
 
 
+
```

Comment:
for figuring out the correct spaces please run pycharm's code >reformat code on all sources (not on the test ones, as that messes up with the fictures)

Reply:
Done - used Reformat and Auto Ident lines on the entire file.